1e/README.md

```diff
@@ -0,0 +1,52 @@
+# 1E
+
+## Overview
+
+The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. You can use these metrics to take advantage of the out-of-the-box dashboard or you can create your own visualizations.
+
+## Setup
+
+### Configuration
+
+Getting set up with the 1E Datadog integration is as simple as emailing a Datadog API key and site to the 1E Support team. There's no further configuration required.
+
+1. Go to your [Datadog Integrations page][1] and click on the 1E tile.
```

Comment:
```suggestion
```
This step doesn't seem to be necessary, because if the user is reading this they'll already be on the tile :)